packages/cozy-harvest-lib/src/components/AccountsPaywall/helpers.js

```diff
@@ -44,12 +44,58 @@ export function computeMaxAccounts() {
   return Infinity
 }
 
+/**
+ * @typedef {Object} AccountCountByKonnector
+ * @property {string} slug - Slug of the konnector
+ * @property {number} count - Number of accounts for the konnector
+ */
+
+/**
+ * @typedef {Object} KonnectorOffer
+ * @property {string} slug - Slug of the konnector
+ * @property {number} offer - Number of accounts offered for the konnector
+ * @property {string} [expiresAt] - Date of expiration of the offer
+ */
+
 /**
  *
- * @param {number} nbAccounts the current number of account for the konnector to check
+ * @param {AccountCountByKonnector[]} accounts the accounts to check
```

Comment:
you can import the IoCozyAccounts type from cozy-client, no,

Reply:
I'm not sure, the cozy-client type doesn't represent the result of the number of accounts per connector. This object only contains a connector slug and its associated number of accounts. Maybe it's the name that's confusing 🤔